tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -179,7 +183,7 @@ describe(__filename, () => {
     const root = render({ addon });
     const link = root.find('.AddonMoreInfo-license-link');
 
-    expect(root.find('.AddonMoreInfo-license-title')).toHaveText('License');
+    expect(root.find('.AddonMoreInfo-license')).toHaveProp('term', 'License');
     expect(link).toHaveProp('children', 'tofulicense');
```

Comment:
I think it's better to do something like:

```js
expect(link.childAt(0)).toIncludeText('tofulicense');
```

but maybe it doesn't matter.

Reply:
I don't think it really matters, but if you prefer that I'll change it, totally cool. 👍